.github/workflows/release.yaml

```
@@ -0,0 +1,80 @@
+on:
+  push:
+    tags:
+      - "*"
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+jobs:
+  build-and-push-image:
```

```suggestion
  build-release-image:
```
Keep "and" if you like. Just swap push to release